docs/sidebars.js

```diff
@@ -10,17 +10,45 @@
  */
 
 module.exports = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
-
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
+  lightdash: [
```

Comment:
smart, nice change!

Reply:
<img width="370" alt="Screenshot 2021-10-05 at 14 22 32" src="https://user-images.githubusercontent.com/11660098/136031429-c4d122c4-4a19-421e-821a-335bf5022938.png">
 is the menu supposed to have double nesting? 
 
 Getting started > Setting up Lightdash > Install Lightdash

Reply:
Yes, you can see the entire menu in the ticket description #502
Right now is weird because we don't have the other pages yet. Going to work on the `Connect a project` next.